Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -134,6 +140,18 @@ extension ModuleError: CustomStringConvertible {
             return "plugin target '\(target)' doesn't have a 'capability' property"
         case .embedInCodeNotSupported(let target):
             return "embedding resources in code not supported for C-family language target \(target)"
+        case .duplicateModules(let package, let otherPackage, let targets):
+            var targetsDescription = "'\(targets.sorted().prefix(3).joined(separator: "', '"))'"
+            if targets.count > 3 {
+                targetsDescription += " and \(targets.count - 3) others"
+            }
+            return "multiple similar targets \(targetsDescription) appear in package '\(package)' and '\(otherPackage)', this may indicate that the two packages are the same and can be de-duplicated by using mirrors. if they are not duplicate consider using the `moduleAliases` parameter in manifest to provide unique names"
```

Comment:
nit: capitalization and line length formatting

```suggestion
            return """
                multiple similar targets \(targetsDescription) appear in package '\(package)' and '\(otherPackage)', \
                this may indicate that the two packages are the same and can be de-duplicated by using mirrors. \
                If they are not duplicate consider using the `moduleAliases` parameter in manifest to provide \
                unique names
                """
```